### PR TITLE
OSM to archive

### PR DESCRIPTION
**Pull request by amye**
This might break, but the project has been voted for archive by TOC

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [ ] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [ ] Have you picked the single best (existing) category for your project?
* [ ] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [ ] Have you added your SVG to `hosted_logos` and referenced it there?
* [ ] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [ ] Does your project/product name match the text on the logo?
* [ ] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [ ] ~30 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by amye** [comment]
:/

**Comment by amye** [comment]
@AndreyKozlov1984  - Ah, we do not have 'archive' defined! Maybe we should

`9:20:41 AM: No relation with name archive`

File: landscape.yml [hunk 1]
```diff
@@ -3957,13 +3957,14 @@ landscape:
...
             homepage_url: https://openservicemesh.io/
-            project: sandbox
+            project: archived
             repo_url: https://github.com/openservicemesh/osm
             logo: open-service-mesh.svg
             twitter: https://twitter.com/openservicemesh
             crunchbase: https://www.crunchbase.com/organization/cloud-native-computing-foundation
             extra:
               accepted: '2020-09-08'
+              archives: '2023-06-30'
```
Comment (caniszczyk) [inline landscape.yml:3967]: isn't this supposed to be 'archived' ?

Comment (amye) [reply landscape.yml:3967]: s is next to d, so yes.

**Pull request merged**